generator/src/cdn/webpack.config.ts

```diff
@@ -10,30 +10,11 @@
 
 import { customizeWebpackConfig } from './webpack-customizer';
 
-const yargs = require('yargs');
```

Comment:
Is this change necessary or just a "drive-by" fix?

Reply:
It's just a dead code, as I understand.
I don't see how it's related to this Webpack config and who calls it.

Reply:
Well, because you removed the options, right. The question is was there a reason to remove the options.

Reply:
ahh.. Got it. I'm reading the options passed to Webpack through the `env` argument of the exported function.
yargs is not needed here anymore, so typescript complains about not used variables